spec/services/noid_spec.rb

```diff
@@ -1,16 +1,9 @@
 require 'spec_helper'
 
 describe Sufia::Noid do
-  describe "#namespaceize" do
-    subject { Sufia::Noid.namespaceize(id) }
-
-    context "when the passed in pid doesn't have a namespace" do
-      let(:id) { 'abc123' }
-      it { is_expected.to eq 'sufia:abc123' }
-    end
-    context "when the passed in pid has a namespace" do
-      let(:id) { 'ksl:abc123' }
-      it { is_expected.to eq 'ksl:abc123' }
-    end
+  describe "#treeify" do
+    subject { Sufia::Noid.treeify(id) }
+    let(:id) { 'abc123def45' }
```

Comment:
Is there a min or max on the number of characters in the noid that Sufia will "treeify" ?